src/controller/python/test_client/scripts/build_sdk.sh

```diff
@@ -0,0 +1,9 @@
+
+# Skip creating environment if it already exists
+if [[ -z "${CHIP_SDK_ROOT}" ]]; then
+  source ./scripts/activate_environment.sh
+fi
```

Comment:
why include this if statement here if it is already in `build_sdk_container.sh`? 

Consider keeping it here and removing this if from `build_sdk_container.sh` if the 
```
gn gen $CHIP_SDK_ROOT/out/debug
ninja -C $CHIP_SDK_ROOT/out/debug
cp $CHIP_SDK_ROOT/out/debug/controller/python/*.whl $TEST_CLIENT_ROOT/
``` 
block will always go together with the if statement highlighted in this comment.

Reply:
> why include this if statement here if it is already in build_sdk_container.sh?

The reason it exists in both places is so we have the freedom to build just the sdk, just the container or build both sdk and the containers, depending on how you are testing the .whl

Reply:
Please create a separate `create_sdk_enviroment.sh` file for just the 
```
# Skip creating environment if it already exists
if [[ -z "${CHIP_SDK_ROOT}" ]]; then
  source ./scripts/activate_environment.sh
fi
```
part so as to avoid duplicate code then.

Reply:
LGTM